core/src/main/scala/org/apache/spark/storage/BlockManagerMessages.scala

```diff
@@ -116,4 +121,11 @@ private[spark] object BlockManagerMessages {
   case class BlockManagerHeartbeat(blockManagerId: BlockManagerId) extends ToBlockManagerMaster
 
   case class HasCachedBlocks(executorId: String) extends ToBlockManagerMaster
+
+  case class GetCachedBlocks(executorId: String) extends ToBlockManagerMaster
+
+  case class GetSizeOfBlocks(blocks: Seq[(String, BlockId)]) extends ToBlockManagerMaster
+
+  case class ReplicateOneBlock(executorId: String, blockId: BlockId, exclude: Seq[String])
```

Comment:
`exclude` should at least be consistent with the name of the corresponding argument in `ReplicateBlock`.

Reply:
fixed